entrypoint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+CLOUD_PROVIDER=${1:-$CLOUD_PROVIDER}
+CRI_RUNTIME_ENDPOINT=${CRI_RUNTIME_ENDPOINT:-/run/peerpod/cri-runtime.sock}
+
+if [[ -S ${CRI_RUNTIME_ENDPOINT} ]]; then
+	CRI_RUNTIME_ENDPOINT_OPTION="-cri-runtime-endpoint ${CRI_RUNTIME_ENDPOINT}"
+fi
+
+test_vars() {
+        for i in $@; do
+                [ -z ${!i} ] && echo "\$$i is NOT set" && EXT=1
+        done
+        [[ -n $EXT ]] && exit 1
+}
+
+aws() {
+#test_vars AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_REGION
+set -x
+cloud-api-adaptor-aws aws \
+	-aws-access-key-id ${AWS_ACCESS_KEY_ID} \
+	-aws-secret-key ${AWS_SECRET_ACCESS_KEY} \
+	-aws-region ${AWS_REGION} \
+	-pods-dir /run/peerpod/pods \
+	${CRI_RUNTIME_ENDPOINT_OPTION} \
+	-socket /run/peerpod/hypervisor.sock
+}
+
+libvirt() {
+test_vars LIBVIRT_URI
+set -x
+cloud-api-adaptor-libvirt libvirt \
+	-uri ${LIBVIRT_URI} \
+	-data-dir /opt/data-dir \
+	-pods-dir /run/peerpod/pods \
+	-network-name ${LIBVIRT_NET:-default} \
+	-pool-name ${LIBVIRT_POOL:-default} \
+	${CRI_RUNTIME_ENDPOINT_OPTION} \
+	-socket /run/peerpod/hypervisor.sock
+}
```

Comment:
@snir911 can you please add ibmcloud option as well
```
/opt/confidential-containers/bin/cloud-api-adaptor-ibmcloud ibmcloud \
    -api-key "${IBMCLOUD_API_KEY}" \
    -key-id "${SSH_KEY_ID}" \
    -image-id "${IMAGE_ID}" \
    -profile-name "${INSTANCE_PROFILE}" \
    -zone-name ${IBMCLOUD_ZONE} \
    -primary-subnet-id "${VPC_PRIMARY_SUBNET_ID}" \
    -secondary-subnet-id "${VPC_SECONDARY_SUBNET_ID}" \
    -primary-security-group-id "${VPC_PRIMARY_SECURITY_GROUP_ID}" \
    -secondary-security-group-id "${VPC_SECONDARY_SECURITY_GROUP_ID}" \
    -vpc-id "${VPC_ID}" \
    -pods-dir /run/peerpod/pods \
    -socket /run/peerpod/hypervisor.sock \
    -cri-runtime-endpoint "${CRI_RUNTIME_ENDPOINT}"
```

Reply:
We used to have this one, however as IBM ppl don't have cycles to test ATM, i discussed with @huoqifeng to split the ibmcloud to a different PR which he'll test later, I'll do this later